presto-druid/src/main/java/com/facebook/presto/druid/DruidQueryGeneratorContext.java

```diff
@@ -168,7 +168,7 @@ public DruidQueryGeneratorContext withAggregation(
                     String definition = entry.getValue().getDefinition();
                     int start = definition.indexOf("(");
                     int end = definition.indexOf(")");
-                    String countDistinctClause = "count ( distinct " + definition.substring(start + 1, end) + ")";
+                    String countDistinctClause = "count ( distinct \"" + definition.substring(start + 1, end) + "\")";
```

Comment:
create a private function to add quotes to column names. could be used here and below

Reply:
done